public/app/features/alerting/unified/components/rule-viewer/RuleViewer.tsx

```diff
@@ -472,6 +472,7 @@ export const calculateTotalInstances = (stats: CombinedRule['instanceTotals']) =
     .pick([
       AlertInstanceTotalState.Alerting,
       AlertInstanceTotalState.Pending,
+      AlertInstanceTotalState.Recovering,
       AlertInstanceTotalState.Normal,
       AlertInstanceTotalState.NoData,
       AlertInstanceTotalState.Error,
```

Comment:
Could we change this to something like this?

```
const states = Object.keys(AlertInstanceTotalState);
return chain(stats).pick(states).values().sum().value();
```

Unless there's some reason that we don't want to link the enum together with the state view :)